server/logging/pubsub.go

```diff
@@ -0,0 +1,68 @@
+package logging
+
+import (
+	"context"
+	"encoding/json"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+)
+
+type pubsubLogWriter struct {
+	topic  *pubsub.Topic
+	logger log.Logger
+}
+
+func NewPubSubLogWriter(projectId string, topicName string, logger log.Logger) (*pubsubLogWriter, error) {
+	ctx := context.Background()
+
+	client, err := pubsub.NewClient(ctx, projectId)
+	if err != nil {
+		return nil, err
+	}
+
+	topic, err := client.CreateTopic(ctx, topicName)
```

Comment:
Let's check for and handle this error.

Reply:
Turns out this line was wrong - I should've used client.Topic(), which doesn't return an error (it doesn't do any RPCs).